pkg/cmd/pulumi/new.go

```diff
@@ -549,6 +592,7 @@ func newNewCmd() *cobra.Command {
 
 			args.interactive = cmdutil.Interactive()
```

Comment:
```suggestion
```

We're just overwriting this below, so I don't think this line is necessary anymore.